cmd/scale.go

```diff
@@ -654,3 +658,15 @@ func (sc *scaleCmd) printScaleTargetEqualsExisting(currentNodeCount int) {
 		}
 	}
 }
+
+// validateOSBaseImage checks if the OS image is available on the target cloud (ATM, Azure Stack only)
+func (sc *scaleCmd) validateOSBaseImage() error {
+	if sc.containerService.Properties.IsAzureStackCloud() {
```

Comment:
I would prefer we put this "if Azure Stack" guard around its invocation (inside the run method above) instead of inside this method itself.